src/selectedtests/test_mappings/create_test_mappings.py

```diff
@@ -1,65 +1,83 @@
 """Test Mappings class to create test mappings."""
-import structlog
 import os.path
+import structlog
 
-from datetime import datetime
-from collections import defaultdict
-from typing import Pattern
-from git import Repo
 from tempfile import TemporaryDirectory
+from typing import Pattern, Tuple
+from collections import defaultdict, namedtuple
 from evergreen.api import EvergreenApi
-from evergreen.manifest import ManifestModule
+from git import Repo
+
 from selectedtests.git_helper import init_repo, modified_files_for_commit
-from selectedtests.evergreen_helper import get_evg_project
+from selectedtests.evergreen_helper import get_evg_project, get_evg_module_for_project
 
 LOGGER = structlog.get_logger(__name__)
+TestMappingsResult = namedtuple(
+    "ProjectMappingsResult",
```

Comment:
Should the `namedtuple` have the same name as the variable it is being assigned to?

Reply:
👍 fixed